integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -194,25 +194,35 @@ jobMaster:
 # UFS journal with HDFS example
 # journal:
 #   type: "UFS"
+#   ufsType: "non-local"
 #   folder: "hdfs://{$hostname}:{$hostport}/journal"
 # EMBEDDED journal to /journal example
 # journal:
 #   type: "EMBEDDED"
 #   folder: "/journal"
 journal:
-  type: "UFS" # "UFS" or "EMBEDDED"
-  ufsType: "local" # Ignored if type is "EMBEDDED". "local" or "HDFS"
-  folder: "/journal" # Master journal folder
+  # [ Required arguments ]
+  type: "UFS" # One of "UFS" or "EMBEDDED"
+  folder: "/journal" # Master journal directory or equivalent storage path
+  # [ Conditionally required arguments ]
+  #
+  # [[ UFS-backed journal options (journal.type="UFS") ]]
+  ufsType: "local" # One of "local" or "non-local"
+  #
+  # [[ K8s volume options ]]
+  # - required when using an EMBEDDED journal (journal.type="EMBEDDED")
+  # - required when using a local UFS journal (journal.type="UFS" and journal.ufsType="local")
+  #
```

Comment:
I couldn't figure out a clean "nested comments" so I used brackets to denote the nesting level. Please lmk if you have suggestions

Reply:
what about more hash signs?
```
# Level 1
## Level 2
### Level 3
```